src/View/mainpage/MainPage.py

```diff
@@ -104,10 +105,39 @@ def setup_central_widget(self):
         # Right panel contains the different tabs of DICOM view, DVH, clinical data, DICOM tree
         self.right_panel = QtWidgets.QTabWidget()
 
-        # Add DICOM view to right panel as a tab
+        # Create a Dicom View containing single-slice and 3-slice views
+        self.dicom_view = QStackedWidget()
+
         roi_color_dict = self.structures_tab.color_dict if hasattr(self, 'structures_tab') else None
         iso_color_dict = self.isodoses_tab.color_dict if hasattr(self, 'isodoses_tab') else None
-        self.dicom_view = DicomView(roi_color=roi_color_dict, iso_color=iso_color_dict)
+        self.dicom_view_single = DicomView(roi_color=roi_color_dict, iso_color=iso_color_dict)
+        self.dicom_view_axial = DicomView(roi_color=roi_color_dict, iso_color=iso_color_dict, format_metadata=False)
+        self.dicom_view_sagittal = DicomView(roi_color=roi_color_dict, iso_color=iso_color_dict, slice_view="sagittal")
+        self.dicom_view_coronal = DicomView(roi_color=roi_color_dict, iso_color=iso_color_dict, slice_view="coronal")
+
+        # Rescale the size of the scenes inside the 3-slice views
+        self.dicom_view_axial.zoom = 0.5
+        self.dicom_view_sagittal.zoom = 0.5
+        self.dicom_view_coronal.zoom = 0.5
+        self.dicom_view_axial.update_view(zoom_change=True)
```

Comment:
can you call `update_views()` instead to avoid duplication?

Reply:
Where shall I place that function ? Each object is a separate DicomView instance.